internal/types/lsp.go

```diff
@@ -618,7 +618,8 @@ type ConfigurationParams struct {
 }
 
 type AuthenticationParams struct {
-	Token string `json:"token"`
+	Token  string `json:"token"`
+	ApiUrl string `json:"apiUrl"`
```

Comment:
this may need an `omitifempty`, as it's optional. That way, we don't need to increase the protocol version for this anyway. Although, we may want to do that anyway for all development. before merging new stuff into LS.